src/main/java/bootcamp/model/User.java

```diff
@@ -0,0 +1,62 @@
+package bootcamp.model;
```

Comment:
there are some standard methods that are usually added (overridden) for all POJOs. See if you find what they are.

Reply:
*upvote*